backend/db/utils.py

```diff
@@ -647,7 +650,8 @@ def get_ddl_statements(
       1. For each table: don't do anything if these tables exist & initialized
       2. Add alters to fix data types (although, should really move this stuff to dtypes settings when creating
       dataframe that loads data into db.
-      3. I think it's inserting a second ; at the end of the last statement of a given module"""
+      3. I think it's inserting a second ; at the end of the last statement of a given module
+      4. consider throwing an error if no statements found, either here, or where func is called"""
```

Comment:
## Unrelated idea 2: Throw error when no DDL statements found
Siggie encountered this edge case